src/fields/field.impl.hh

```diff
@@ -313,21 +328,22 @@ void Field<spacedim,Value>::update_history(const TimeStep &time) {
     // read input up to given time
 	double input_time;
     if (shared_->input_list_.size() != 0) {
-        while( shared_->list_idx_ < shared_->input_list_.size()
+        while( shared_->list_idx_ < (int)shared_->input_list_.size()
```

Comment:
Could shared_->list_idx_ be unsigned? It could be initialized to input_list_.size() ==0.